client/start.js

```diff
@@ -157,7 +157,10 @@ export function navigateToPage(dispatch, path, doNotPushPath) {
       });
 
       if (!doNotPushPath) {
-        history.push(pathname);
+        history.push({
+          pathname,
+          search: "",
+        });
         registerPageView(page, pathname);
```

Comment:
history even takes a location-like object, in which you can specify the query string.
- [docs history ](https://github.com/ReactTraining/history/blob/28c89f4091ae9e1b0001341ea60c629674e83627/docs/navigation.md)
- [location object](https://developer.mozilla.org/en-US/docs/Web/API/Location)

Console logging history obj:
![Screenshot 2021-03-11 at 5 51 53 PM](https://user-images.githubusercontent.com/27222520/110787132-0031d000-8293-11eb-9429-f95927ab1196.png)